crates/astria-sequencer-client/Cargo.toml

```diff
@@ -15,8 +15,8 @@ serde_json = { workspace = true }
 tendermint = { workspace = true }
 tendermint-rpc = { workspace = true, features = ["http-client"]}
 tokio = { workspace = true }
+wiremock = "0.5"
```

Comment:
Why is this becoming no longer a dev-dependency?

Reply:
Dev dependency again by rebasing on top of the recent sequencer-client changes.